video_xblock/static/js/studio-edit-utils.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Auxiliary functions for studio editor modal's JS.
+ */
+
+/**
+ * Prepare data to be saved to video xblock.
+ */
+function fillValues(fields) {
```

Comment:
Do all of those function go to global scope in Studio? If so, how can you guarantee lack of naming conflicts?